tags/splunk.md

```diff
@@ -0,0 +1,7 @@
+---
+name: splunk
+displayName: Splunk
+description: A schema-less search engine with very easy to use propietary query language that makes aggregations a seasy as it can get.
```

Comment:
typo on easy and proprietary

```suggestion
description: A schema-less search engine with very easy to use proprietary query language that makes aggregations as easy as it can get.
```